Sources/SwiftASN1/Basic ASN1 Types/PEMDocument.swift

```diff
@@ -53,33 +188,46 @@ public struct PEMDocument {
             throw ASN1Error.invalidPEMDocument(reason: "PEMDocument not correctly base64 encoded")
         }
 
-        self.type = discriminator
-        self.derBytes = derBytes
+        self.discriminator = discriminator
+        self.derBytes = Array(derBytes)
+        #else
+        fatalError("PEM decoding currently not supported without Foundation.")
+        #endif
     }
 
-    public init(type: String, derBytes: Data) {
-        self.type = type
+    public init(type: String, derBytes: [UInt8]) {
+        self.discriminator = type
         self.derBytes = derBytes
     }
 
+    #if canImport(Foundation)
```

Comment:
When is this `canImport` ever false?

Reply:
I don't know. The code was previously guarded by this canImport so I just carried it forward but tried to make as much of the code compiler without access to Foundation.Data. I'm happy to remove it if we no longer need it.

Reply:
Oh no I recall why this is here.

I don't think it makes any sense to have the rest of the code compile without Data, given that it cannot possibly execute. I think it's fine to have these protocols simply missing when this conditional is true.